src/features/author/authorSlice.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
pls refactor your code so that u don't have to turn off this eslint rule
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**